frontend/src/views/WheresMyGene/components/HeatMap/components/XAxisChart/index.tsx

```diff
@@ -69,7 +74,10 @@ export default function XAxisChart({ geneNames }: Props): JSX.Element {
           width={heatmapWidth}
           ref={xAxisRef}
         />
-        <XAxisMask />
+        <div style={{ display: "flex" }}>
```

Comment:
Ideally let's create a styled component for this, so we don't have inline styling 🙏